cmd/anonymizer/app/query/query.go

```diff
@@ -17,7 +17,7 @@ import (
 
 	"github.com/jaegertracing/jaeger-idl/model/v1"
 	"github.com/jaegertracing/jaeger-idl/proto-gen/api_v2"
-	"github.com/jaegertracing/jaeger/storage/spanstore"
+	"github.com/jaegertracing/jaeger/internal/storage/v1/spanstore"
```

Comment:
One thing that bothers me is that both interfaces and implementations are all under v1/ dir. Any thoughts? One option is to stash interfaces under v1/api/*

Reply:
@yurishkuro I was thinking the same thing as well - that sounds good to me